manifests/params.pp

```diff
@@ -75,7 +76,9 @@
     'Debian': {
       # system packages
       $system_packages = ['libicu-dev', 'python2.7','python-docutils',
-                          'libxml2-dev', 'libxslt1-dev','python-dev']
+                          'libxml2-dev', 'libxslt1-dev','python-dev', 'cmake', 'pkg-config', 'libkrb5-dev', 'ruby-execjs']
+                          
+      $redis_packages = ['redis-server']
```

Comment:
Redis should be managed outside of this module. There are many Redis modules available on the Puppet Forge.